devel/ld64/Portfile

```diff
@@ -288,7 +306,7 @@ if {${subport} eq ${name}} {
     # dependency cycle as llvm requires apple-gcc42 and ld64 to build correctly.  Users
     # wanting LTO support in ld64 on Tiger can install the +llvm variant after llvm
     # has been installed.
-
+    
```

Comment:
This change adds trailing whitespace, which the portfile currently does not have. I'd revert this.

Reply:
Yep, slip of the tab. I'll fix it if Jeremy accepts this PR in general.